services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable no-console */
+// import AWS from 'aws-sdk';
+import to from 'await-to-js';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import uuid from 'uuid';
+
+import config from '../../../config';
+import params from '../../../libs/params';
+import hash from '../../../libs/helperHashEncode';
+import * as request from '../../../libs/request';
+import { putItem } from '../../../libs/queries';
+import * as dynamoDB from '../../../libs/dynamoDb';
+
+const CASE_STATUS_OPEN_TO_APPLY = 'openToApply';
+const CASE_PROVIDER_VIVA = 'VIVA';
+const VADA_SSM_PARAMS = params.read(config.vada.envsKeyName);
+const CASE_SSM_PARAMS = params.read(config.vada.envsKeyName);
```

Comment:
Lägg med fördel in konstanter i filen constants.js som du finner i mappen libs.
På så vis håller vi kodbasen lite mer strukturerad.